mini_stitch.gemspec

```
@@ -32,7 +32,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'rake', '~> 12.2.1'
   spec.add_development_dependency 'rspec', '~> 3.7.0'
 
-  spec.add_dependency 'json', '>= 1.8', '<2.0'
+  spec.add_dependency 'json', '~> 2.6.1'
```

We need a not-so-strict bump here.

```suggestion
  spec.add_dependency 'json', '>= 1.8', '< 3.0'
```

We have to do this way because this gem is public and someone out there may be using this gem - it's published on Stitch website as one gem people can use to interact with their product..

---

ah ok yep that makes sense.